core/src/test/scala/sttp/client4/testing/streaming/StreamingTest.scala

```diff
@@ -208,6 +221,7 @@ abstract class StreamingTest[F[_], S]
 
         urlRegex.findAllIn(responseBody).length shouldBe numChunks
       }
+    retryImmediatelyOnError(runTest, retriesLeft = 5)
```

Comment:
sorry for commenting on merged PRs, but I'd add a comment here why this particular tests is "fixed" by adding a repeat :)